docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express')
+const { HTTPReceiver } = require('cloudevents-sdk')
```

Comment:
You can also get a default-to-1.0 `CloudEvent` here.

```js
const { HTTPReceiver, CloudEvent } = require('cloudevents-sdk');
```

There is a [WIP pull request](https://github.com/cloudevents/sdk-javascript/pull/141) currently that will also add `HTTPEmitter` to this. If you can hang tight, I hope to land that this week.